card/mod/05_standard/set/all/rich_html/modal.rb

```diff
@@ -37,7 +37,7 @@ def default_modal_content_args args
   end
 
   view :modal_body do |args|
-    _render_content(args)
+    _render_core(args)
```

Comment:
will merge.  why is there a separate view for this?

Also, I think you can just do
````
  view :modal_body, :core
````
...for the same effect